kratos/tests/containers/test_data_value_container.cpp

```diff
@@ -38,5 +42,50 @@ KRATOS_TEST_CASE_IN_SUITE(DataValueContainer, KratosCoreFastSuite) {
   for (std::size_t i = 0; i < distances.size(); i++)
     KRATOS_CHECK_EQUAL(distances[i], original_distances[i]);
 }
+
+/**
+ * This test checks recursively the proper functioning of the DataValueContainer
+ */
+KRATOS_TEST_CASE_IN_SUITE(DataValueContainerRecursive, KratosCoreFastSuite) {
```

Comment:
why recursive? I don't see any recursion here....

Reply:
Repeated may be better, recursive reflects to me as calling the function inside itself...

Meanwhile I don't think these stress tests are useful at all....